controls/V-72013.rb

```diff
@@ -44,13 +24,14 @@
 
 If the value for \"CREATE_HOME\" parameter is not set to \"yes\", the line is
 missing, or the line is commented out, this is a finding."
-  tag "fix": "Configure the operating system to assign home directories to all new
-local interactive users by setting the \"CREATE_HOME\" parameter in
+  tag "fix": "Configure the operating system to assign home directories to all
+new local interactive users by setting the \"CREATE_HOME\" parameter in
 \"/etc/login.defs\" to \"yes\" as follows.
 
 CREATE_HOME yes"
+  tag "fix_id": "F-78365r1_fix"
 
   describe login_defs do
-    its('CREATE_HOME') { should eq 'yes' }
+    its('CREATE_HOME') { should match %r{yes}i }
```

Comment:
The previous version of this should be reinstated.

The `man` page does not appear to allow for `YES` (though it may work) and this also allows for `noyes`, `yes indeed`, etc...